### PR TITLE
docs: add ai scan command for cluster scan

### PR DESCRIPTION
**Pull request by vaijab**
## Description

Part of https://github.com/kubeshop/botkube-cloud/issues/1062

Changes proposed in this pull request:

- docs: add ai scan command for cluster scan

### CODE REVIEW

File: docs/plugins/ai-assistant.mdx [hunk 1]
```diff
@@ -69,6 +69,11 @@ Here are a few examples to get you started:
 - `@Botkube ai what Botkube plugins are available to install? List all of them with one sentence description and link to the docs.`
 - `@Botkube ai who's the Botkube maintainer?`
```
Comment (pkosiec) [inline docs/plugins/ai-assistant.mdx:71]: Could you also list the cluster scan feature in the `## Features` section?

**Review by pkosiec — APPROVED** [review]
I think we can sync the docs to the 1.11. As there are no breaking changes, we can release the new AI plugin for 1.11 too 👍 But we can also do it later, up to you.

**Comment by vaijab** [comment]
All suggestions addressed @pkosiec

**Review by pkosiec — APPROVED** [review]
LGTM 👍

**Pull request merged**